src/components/header.component/Header.jsx

```diff
@@ -0,0 +1,66 @@
+import "./header.css";
+export const Header = () => (
+  <header className="nav-header  nav-header--e-comm nav--e-comm-bg  w-p-full">
+    <nav className="nav nav--e-comm  nav--e-comm-bg flex-r flex-s flex-sp-eve ">
+      {/* <!-- Brand Name  --> */}
+      <a
+        className=" brand-name--e-comm text-md m-lr-md "
+        href="../../index.html"
+      >
+        PHOENIX<sub>SHOP</sub>
+      </a>
+      {/* <!-- Search Field  --> */}
+      <label className="nav-search flex-r br-md p-xs ">
+        <img
+          src="https://ui-phoenix.netlify.app/Assets/svg/navigation/search.svg"
+          alt="search"
+        />
+        <input
+          className="nav-input m-lr-md w-p-full h-p-full "
+          type="text"
+          placeholder="Search"
+        />
+      </label>
+      {/* <!-- RHS Icons  --> */}
+      <ul className="nav-menu__items flex-r flex-center">
+        <li>
+          <button className="button button--secondary login-btn">
+            <a href="./log-in.html">Login</a>
+          </button>
+        </li>
+        <li>
+          <a
+            className="color-white text-sm"
+            href="../product-listing/product-listing.html"
+          >
+            SHOP NOW{" "}
```

Comment:
Remove the empty jsx component from this line

Reply:
Thanks ! Will do that.